tests/test_alembic.py

```diff
@@ -0,0 +1,32 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+import unittest
+
+from alembic.config import Config
+from alembic.script import ScriptDirectory
+
+
+class TestAlembic(unittest.TestCase):
+
+    def test_only_single_head_revision_in_migrations(self):
```

Comment:
Maybe move this test to https://github.com/apache/airflow/blob/master/tests/utils/test_db.py

This file contains a closely related test, to check if the schema of the database, and the schema in the python files are in sync.

Reply:
Maybe also add a small comment on when we could have two heads, and how to resolve it :-)

Reply:
I think this would be better to have in the PR template?

Reply:
I think the PR template is already pretty bloated :) (I think we should somehow simplify this). I found it initially useful, but then it becomes more and more tiresome. Let's not make it more complex.

I think everything that is now in the template should be automatically verified by the checks in CI and commiters should not even think about checking it.

Reply:
Agree with Fokko -> better have less test files.

Reply:
> I think the PR template is already pretty bloated :) (I think we should somehow simplify this). I found it initially useful, but then it becomes more and more tiresome. Let's not make it more complex.

Totally agree. Would be nice if we the template changes depending on the files that changed.

Reply:
Can you rebase onto master and move this test to the other file? :-)

Reply:
Yes will do very shortly was out for a bit

Reply:
moved test.  added small comment, just pointing user to 'alembic merge' command.